### ISSUE TITLE
grunt-devtools should be a standalone util

### ISSUE THREAD

**Issue by cowboy** [vladikoff/grunt-devtools#3]
This is a great idea, but it should be a standalone utility.

Right now, if someone wants to incorporate grunt-devtools into their project, it's via Grunt plugin that goes in the package.json / Gruntfile. They then amend their Readme to tell other collaborators how to use their sexy new grunt-devtools workflow.

Except not everyone who contributes to a project will want to or be able to use grunt-devtools. Someone might be on an unsupported OS or not have access to Chrome or might just not want to use it because they prefer a text-based shell. For those users, this project-proscribed GUI around Grunt can actually hinder collaborative development.

I'd like to suggest that you make grunt-devtools a standalone utility, instead of a Grunt plugin, that is _not_ installed locally into a project, because it exists orthogonal to projects and is more developer-centric.

Maybe it could be made to work in such a way that a potential contributor reading a Readme written by someone using either CLI-based Grunt or grunt-devtools can read one set of directions and understand how it works in whichever mode they are more familiar.

Basically, make it like a shell, but with buttons and progress bars. But please don't make it a Grunt plugin.

**Comment by raphschmidt** [comment]
I don't understand the reason behind this request. grunt-devtools doesn't prevent you from using grunt from the cli. What prevents one set of instructions from being applicable to both grunt and grunt-devtools?

**Comment by vladikoff** [comment]
`0.2.0` will have a way of installing `grunt-devtools` globally. This address points that are part of this issue.
Currently in https://github.com/vladikoff/grunt-devtools/tree/dev 

Now users can either use 
`grunt-devtools` Globally 
or 
`grunt devtools` Locally

**Comment by jgallen23** [comment]
any idea when this will be released?
